src/pages/Cart/ProductPrice.js

```diff
@@ -0,0 +1,29 @@
+// 구매하기 버튼
+const buyProduct = () => {};
+
+const ProductPrice = ({ priceAll }) => {
+  priceAll = priceAll.toLocaleString();
+
+  return (
+    <aside className="orderPrice">
+      <article>
+        <header>
+          <h3>주문 예정 금액</h3>
+        </header>
+        <dl>
+          <dt>총 상품 금액</dt>
+          <dd>{priceAll}</dd>
```

Comment:
```suggestion
const ProductPrice = ({ priceAll }) => {
  return (
    <aside className="orderPrice">
      <article>
        <header>
          <h3>주문 예정 금액</h3>
        </header>
        <dl>
          <dt>총 상품 금액</dt>
          <dd>{priceAll.toLocaleString()}</dd>
```
가급적 값을 재할당하지 않는 것이 좋습니다!